client/src/App.js

```diff
@@ -61,30 +56,15 @@ class App extends Component {
         <div className="container-fluid MainPage">
           <div className="row">
             <div className="col-md-12">
-              <Switch>
-                <Route
-                  exact path="/"
-                  component={Landing} />
-                {/* <Route exact path="/home" component={Home} /> */}
-                <Route
-                  path="/signup"
-                  render={() =>
-                    <SignUp
-                      signup={this.signup}
-                    />}
-                />
-                <Route
-                  path="/signin"
-                  render={() =>
-                    <SignIn
-                      updateUser={this.updateUser}
-                    />}
-                />
-              </Switch>
+              <Router>
+                <Landing path="/" />
+                <SignUp path="/signup" signup={this.signup} />
+                <SignIn path="/signin" updateUser={this.updateUser} />
+              </Router>
```

Comment:
@stuartt this is a test of a public repo comment

Reply:
Got it.